cpp/test/resources/torchscript_model/babyllama/babyllama_handler/config.json

```diff
@@ -0,0 +1,5 @@
+{
+"checkpoint_path" : "/home/ubuntu/serve/cpp/stories15M.bin",
```

Comment:
Would be good to make these relative instead of absolute.

Reply:
@mreso 

```
-rw-rw-r-- 1 ubuntu ubuntu 424K Sep  5 23:55 tokenizer.bin
-rw-rw-r-- 1 ubuntu ubuntu 58M Jul 27 04:09 /home/ubuntu/Downloads/stories15M.bin
```

These two files are needed at run time for the unit test case. I can think of two apporaches
1. We can download these files in `build.sh` and remove it once the unit test case passes
2. Add the download logic in cpp -  `serve/cpp/test/backends/torch_scripted/torch_scripted_backend_test.cc` when executing the test case.

Second option seems to be more resonable. Whats your opinion ?